UnityProject/Assets/Scripts/Systems/Spells/Wizard/SpawnByPortal.cs

```diff
@@ -52,6 +52,14 @@ public static void AnimatePortal(GameObject portal, PortalSpawnInfo settings)
 		public static void AnimateObject(GameObject entity, PortalSpawnInfo settings)
 		{
 			Transform spriteObject = entity.transform.Find("Sprite");
```

Comment:
what does this do?

Reply:
Finds the Sprite sub-gameobject so it can animate it. Seems to be glitchy if you use root gameobject, probably fighting with CustomNetTransform or something.